src/Jackett.Common/Indexers/Milkie.cs

```diff
@@ -0,0 +1,167 @@
+        {
+            LoadValuesFromJson(configJson);
+
+            try
+            {
+                var results = await PerformQuery(new TorznabQuery());
+
+                if (results.Count() == 0)
+                    throw new Exception("Testing returned no results!");
+
+                IsConfigured = true;
+                SaveConfig();
+                return IndexerConfigurationStatus.Completed;
+            }
+            catch (Exception e)
+            {
+                IsConfigured = false;
+                throw new ExceptionWithConfigData(e.Message, configData);
+            }
+        }
+
+        protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
+        {
+            var queryParams = new NameValueCollection
+            {
+                { "ps", "100" }
+            };
+
+            if (!string.IsNullOrWhiteSpace(query.SearchTerm))
+            {
+                queryParams.Add("query", query.SearchTerm);
+            }
+
+            if (query.HasSpecifiedCategories)
+            {
+                queryParams.Add("categories", string.Join(",", MapTorznabCapsToTrackers(query)));
+            }
+
+            var endpoint = TorrentsEndpoint + "?" + queryParams.GetQueryString();
+            var headers = new Dictionary<string, string>() { { "x-milkie-auth", configData.Key.Value } };
+            var jsonResponse = await RequestStringWithCookies(endpoint, null, null, headers);
+
+            var releases = new List<ReleaseInfo>();
+
+            try
+            {
+                var response = JsonConvert.DeserializeObject<MilkieResponse>(jsonResponse.Content);
+
+                var dlQueryParams = new NameValueCollection
+                {
+                    { "key", configData.Key.Value }
+                };
+
+                foreach (var torrent in response.Torrents)
+                {
+                    var link = new Uri($"{TorrentsEndpoint}/{torrent.Id}/torrent?{dlQueryParams.GetQueryString()}");
+                    var comments = new Uri($"{SiteLink}browse/{torrent.Id}");
+
+                    var release = new ReleaseInfo()
+                    {
+                        Title = torrent.ReleaseName,
+                        Link = link,
+                        Comments = comments,
+                        Guid = comments,
+                        Size = torrent.Size,
+                        Category = MapTrackerCatToNewznab(torrent.Category.ToString()),
+                        Seeders = torrent.Seeders,
+                        Peers = torrent.Seeders + torrent.PartialSeeders + torrent.Leechers,
+                        Grabs = torrent.Downloaded,
+                        UploadVolumeFactor = 1,
+                        DownloadVolumeFactor = 0,
+                        MinimumRatio = 1,
+                        MinimumSeedTime = 172800,
+                        PublishDate = DateTimeUtil.FromUnknown(torrent.CreatedAt)
```

Comment:
temporarily assign result of `DateTimeUtil.FromUnknown` to variable so we can get a line number if exception is thrown.